Duplicati/Library/Utility/ThrottledStream.cs

```diff
@@ -147,24 +147,25 @@ public override int Read(byte[] buffer, int offset, int count)
         }
 
 		/// <summary>
-		/// Write the specified buffer, offset and count.
+		/// Write from a buffer to this stream.
 		/// </summary>
-		/// <param name="buffer">The buffer to write to.</param>
+		/// <param name="buffer">The buffer to read from.</param>
 		/// <param name="offset">The offset into the buffer.</param>
 		/// <param name="count">The number of bytes to write.</param>
         public override void Write(byte[] buffer, int offset, int count)

```

Comment:
This method uses a mixture of spaces and tabs for the indentation.  I decided to not fix the indentation to keep the diff here cleaner looking.